customerio/customerio-expo-plugin
# fix: xcode race condition fix

Resolves https://github.com/customerio/customerio-expo-plugin/issues/72 
Partially resolves: https://github.com/customerio/issues/issues/10195
Build on the changes in https://github.com/customerio/customerio-expo-plugin/pull/73

## Changes
- Used `withXcodeProject` to modify `project.pbxproj` instead of `xcode` package

## Testing instruction
- Install the plugin
- run rebuild and then build (on local and EAS) and run
- Make sure push rich push and all other features work as extected
